contents/uix/accessibility.md

```diff
@@ -0,0 +1,137 @@
+   * older people with changing abilities due to ageing
+   * people with “temporary disabilities” such as a broken arm or lost glasses
+   * people with “situational limitations” such as in bright sunlight or in an environment where they cannot listen to audio
+   * people using a slow Internet connection, or who have limited or expensive bandwidth
+
+## How can I start improving accessibility?
+
+There are many ways to improve accessibility of your website. Below are some tips, divided into the type of disabilities they address.
+
+### Visual
+Users with visual impairments rely on assistive technologies such as a magnifier or a screen reader.
+
+  * Use more visual indicators to convey a message.
+  
+  Let us look at Facebook sign up page. Suppose I want to sign up for a new account, but I have not put in all the necessary information. So, it is supposed to tell me that my attempt to sign up failed.
+
+  This page below uses a red color border around the text box to show that the information needed is missing. As red usually signifies failure, this seems enough.
+  ![Facebook sign up page](https://i.imgur.com/HJPD8Fu.png)
+
+  However, to someone with color blindness, this is how the page looks like
+  ![Facebook sign up page to someone with achromatopsia](https://i.imgur.com/lsI5YI8.png)
+  _[Achromatopsia](https://en.wikipedia.org/wiki/Achromatopsia)_
+
+  ![Facebook sign up page to someone with deuteranopia](https://i.imgur.com/uC9kcTb.png)
+  _[Deuteranopia](https://en.wikipedia.org/wiki/Deuteranopia)_
+
+  It is unclear to color-blind users that the sign up has failed. To avoid confusion, we should not rely on colours alone to convey a message. Instead, we should use more visual indicators such as icons or an explanation box.
+  ![Sign up page with more visual indicators](https://i.imgur.com/jA0cfKd.png)
+
+  * Maintain good contrast
+  
+  Look at this page taken from Tech Crunch.
+  ![Tech Crunch page](https://i.imgur.com/sgBuXNL.png)
+
+  To users with good vision, the design may look minimalist and clean. However, to someone suffering from cataract, the page looks like this.
+
+  ![Blurry Tech Crunch page](https://i.imgur.com/QapHc6t.png)
+  It gets difficult to read the news snippets because of the poor color contrast (grey on white). On the other hand, the black colored texts are still legible. Thus, we should maintain good contrast ratio in our websites for ease of reading.
+
+  * Use `alt` attribute for images
+  
+  The `alt` attribute provides alternative information for an image that can be read out by a screen reader to describe an image.
+
+  ![Use of alt tag](https://i.imgur.com/Jted6hW.png)
+
+  
+### Auditory
+Hearing-impaired users do use assistive technologies such as a hearing aid or a cochlear implant, but these are not specific for accessing websites.
+
+For users with hearing impairment, we should provide text alternatives to audio content, such as:
```

Comment:
I personally feel this section is stating the obvious.

Reply:
Yeah, it is quite straightforward because audio content isn't as commonly found as text-based content. I roughly follow the article here: https://webaim.org/articles/auditory/